index.js

```diff
@@ -23,3 +23,5 @@ app.on('error', error => {
 });
 
 app.start();
+
+console.log(`Listening to queue at ${config.sqs.url}`);
```

Comment:
Is this intentionally left in?

Reply:
Yes. It was annoying me that every other service logged a line when it started except this one.